modules/labs-react/expandable/stories/Expandable.stories.mdx

```diff
@@ -83,82 +82,4 @@ the example below, we're hoisting the models to expand and collapse all three co
 
 ## Components
 
-### Expandable
-
-#### Usage
-
-`Expandable` wraps an `Expandable.Target` and an `Expandable.Content`. By default, it provides a
-`DisclosureModel` for its sub components. Alternatively, a model may be passed in using the hoisted
-model pattern.
-
-#### Props
-
-<ArgsTable of={Expandable} />
-
-### Expandable.Target
-
-#### Usage
-
-`Expandable.Target` creates a heading and a button. The heading is a semantic heading to describe
-the associated content. The button provides users the ability to toggle the associated content.
-
-As according to the
-[W3 disclosure specification](https://www.w3.org/TR/wai-aria-practices-1.1/#disclosure), the button
-has `aria-expanded` and `aria-controls` attributes set by default
-
-This component should hold an `Expandable.Icon`, an optional `Expandable.Avatar`, and an
-`Expandable.Title`.
-
-#### Props
-
-`headingLevel` is the only prop spread to the underlying heading. All other props are spread to the
-button.
-
-<ArgsTable of={Expandable.Target} />
-
-### Expandable.Title
-
-#### Usage
-
-`Expandable.Title` styles the target text that describes the content.
-
-#### Props
-
-<ArgsTable of={Expandable.Title} />
-
-### Expandable.Icon
-
-#### Usage
-
-`Expandable.Icon` creates an icon to visually indicate the state of the content. It takes an
-`iconPosition` prop to determine which chevron icon to use.
-
-#### Props
-
-`iconPosition` takes a value of either `start` or `end`. If no value is provided, it will default to
-`start`. Given a value of `start` it will use a chevron that points down when the container is
-expanded and right when the container is collapsed. Given a value of `end`, it will use a chevron
-that points up when the container is expanded and down when the container is collapsed.
-
-<ArgsTable of={Expandable.Icon} />
-
-### Expandable.Avatar
-
-#### Usage
-
-`Expandable.Avatar` is an optional component that creates an `Avatar` to display a decorative image.
-
-#### Props
-
-<ArgsTable of={Expandable.Avatar} />
-
-### Expandable.Content
-
-#### Usage
-
-`Expandable.Content` holds the content that will be conditionally expanded and collapsed. It has an
-`id` to ensure the `Expandable.Target` properly set it to the `aria-controls` attribute.
-
-#### Props
-
-<ArgsTable of={Expandable.Content} />
+<SymbolDoc name="Expandable" hideDescription />
```

Comment:
FYI, something I noticed when checking the tail end of the doc, additional properties shows up with an empty object.

<img width="943" alt="Screenshot 2023-02-26 at 8 55 53 PM" src="https://user-images.githubusercontent.com/1075861/221470865-1048cdb2-14db-4e1d-aad7-5e5805d2b928.png">

Reply:
I made the "Additional Properties" section optional. If there are no properties, this section will not render